src/Server.hs

```diff
@@ -0,0 +1,29 @@
+{-# LANGUAGE DataKinds     #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE RankNTypes    #-}
+{-# LANGUAGE TypeOperators #-}
+
+module Server (api, app) where
+
+import Data.Proxy                 (Proxy (..))
+
+import Servant
+
+import Servant.API.Generic
+import Servant.Server.Generic
+
+data Routes route = Routes {
+    _get :: route :- "api" :> "the-answer" :> Get '[JSON] Int
+    }
+    deriving (Generic)
+
+api :: Proxy (ToServantApi Routes)
+api = genericApi (Proxy :: Proxy Routes)
```

Comment:
this variable looks unused.
Usually you need to create the data type instead.

```haskell
type HintmanAPI = ToServantApi HintmanSite
```

As you can notice the naming for that is: data type name + `API` suffix.